src/views/penaltydetails.njk

```diff
@@ -0,0 +1,68 @@
+{% extends "layout.njk" %}
+
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/back-link/macro.njk"  import govukBackLink %}
+{% from "govuk/components/input/macro.njk"      import govukInput %}
+{% from "govuk/components/details/macro.njk"    import govukDetails %}
+{% from "govuk/components/button/macro.njk"     import govukButton %}
+
+
+{% block pageTitle %}
+  Tell us the penalty details
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        {{ govukErrorSummary
+          ({
+            titleText: "There is a problem with the details you gave us",
+            errorList: validationResult.errors
+          }) if validationResult and validationResult.errors.length > 0 
+        }}
+      <h1 class="govuk-heading-xl">
+        What are the penalty details?
+      </h1>
+      <form method="post" action="">
+        {{ 
+          govukInput({
+            label: {
+              text: "Company number"
+            },
+            hint: {
+              text: "This is the 8-character reference issued by Companies House when the company was set up."
+            },
+            errorMessage: validationResult.getErrorForField('companyNumber') if validationResult,
+            classes: "govuk-input--company-number",
+            id: "company-number",
+            name: "companyNumber",
+            value: companyNumber
+          }) 
+        }}
+        {{ 
+          govukInput({
+            label: {
+              text: "Reference number"
+            },
+            hint: {
+              text: "This is the 9-character reference shown on the penalty notice."
+            },
+            errorMessage: validationResult.getErrorForField('penaltyReference') if validationResult,
+            classes: "govuk-input--penalty-reference",
+            id: "penalty-reference",
+            name: "penaltyReference",
+            value: penaltyReference
+          }) 
+        }}
+        {{ 
+          govukButton({
+            text: "Continue",
```

Comment:
We are not consistent in terms of using single vs double quotes. At some point we should settle on one of these. Not necessarily problem in the context of this story.

Reply:
my editor enforces single quotes in typescript files, not in njks (which are mostly copied from the prototype). But I think we are generally in agreement regarding single quotes